CHANGELOG.md

```diff
@@ -7,18 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [vNext]
 
-- ...
+## [4.5.0]
+
+### Changed
+
+- Updated Substrate to polkadot-v0.9.27
 
 ## [4.4.0]
 
 ### Changed
 
 - Updated Substrate to polkadot-v0.9.26
+- New `cluster` parameter for `serve` and `store` calls in `pallet-ddc-staking` to specify the DDC cluster ID which the caller is willing to join
```

Comment:
Reading the changelog it looks like the change may affect mainnet users while it actually does not when we only add the change to the devnet runtime :thinking:

Reply:
I don't mean we should change something right now, just sharing my thought.

Reply:
Next time, let's specify which runtime the change relates to.